### PR TITLE
Added heap sort

### PR DESCRIPTION
**Pull request by antrix190**
Changes made for issue #2 : 
1. Added Heap Sort
2. Moved java class files to default package.

### CODE REVIEW

File: Dijkstra.java [hunk 1]
```diff
@@ -1,8 +1,3 @@
-/*
-this program is used to find minimum distance between two vertices while traversing all available vertices.
-*/
-package javacodes;
```
Comment (181514912) [inline Dijkstra.java]: Here comments should not have been removed

**Comment by 181514912** [comment]
Thanks @antrix190 for your contributions. It would have been great if comments where not removed. Try it to be able to merge.

**Comment by antrix190** [comment]
@181514912 I'll add it again

**Comment by antrix190** [comment]
@181514912 Issue has been resolved. Please check and merge.

**Comment by 181514912** [comment]
Thanks. All fine 👍

**Pull request merged**